docs/_static/Makefile

```diff
@@ -1,5 +1,7 @@
 .PHONY : all
 
+INKSCAPE := $(shell command -v inkscape 2> /dev/null)
```

Comment:
I've (hopefully) made it so that if you make the docs without inkscape, it'll still work.

Reply:
Thanks for working on that. Hopefully it benefits others besides me

Reply:
My inkscape is the wrong version and I got an error trying to build the docs related to the PNG flag. So, I think this is probably just MY issue and not an issue with the Makefile. Just wanted to let you know in case this is useful

My inkscape help menu doesn't have "export-type", but says for PNG:
  -e, --export-png=FILENAME                  Export document to a PNG file